packages/workbox-precaching/utils/removeIgnoredSearchParams.mjs

```diff
@@ -0,0 +1,33 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+import '../_version.mjs';
+
+/**
+ * Removes any URL search parameters that should be ignored.
+ *
+ * @param {URL} urlObject The original URL.
+ * @param {Array<RegExp>} ignoreUrlParametersMatching RegExps to test against
+ * each search parameter name. Matches mean that the search parameter should be
+ * ignored.
+ * @return {URL} The URL with any ignored search parameters removed.
+ *
+ * @private
+ * @memberof module:workbox-precaching
+ */
+export default function(urlObject, ignoreUrlParametersMatching) {
```

Comment:
Same comment as above, can we make this a named export?

Reply:
The code that calls this module is inside a `NODE_ENV` check.

Reply:
Changed.